doc/recipe/overview.rst

```diff
@@ -90,11 +90,18 @@ data specifications:
   Please note that `yaml`_ interprets numbers with a leading ``0`` as octal numbers,
   so we recommend to avoid them. For example, use ``128`` to specify the year
   128 instead of ``0128``.)
-  Alternatively, the time range can be specified in `ISO 8601 format <https://en.wikipedia.org/wiki/ISO_8601>`_, for both dates
-  and periods. Or as a wildcard to work with all available data, set the starting
-  point at the first available year, and set the ending point at the last available
-  year. The starting point and end point must be separated with '/'.
-  (e.g key-value ``timerange: '1982/1990'``)
+  Alternatively, the time range can be specified in `ISO 8601 format
+  <https://en.wikipedia.org/wiki/ISO_8601>`_, for both dates and periods.
+  The starting point and end point must be separated with ``/`` (e.g.,
+  ``timerange: '1982/1990'`` will select the data from the years 1982 to 1990,
+  ``timerange: '1910/P2Y'`` will select 2 years of data starting in 1910, or
+  ``P1Y2M/19201231T235959`` will select 1 year and 2 months of data ending on
+  31 December 1920 23:59:59).
+  In addition, wildcards (``'*'``) are accepted, which allow the selection of
```

Comment:
The combination of timerange values, wildcards and more examples is explained starting in line 167

Reply:
Ahh, right, didn't see that! I will revert this change and add a pointer to this section